tools/install.sh

```diff
@@ -9,6 +9,12 @@
 
 set -euo pipefail
 
+# Script must be run from tools directory.
+if [[ ${PWD:(-6)} != *'/tools'* ]]; then
```

Comment:
```suggestion
if [[ "$PWD" != *'/tools' ]]; then
```

Can you see if this works?

Reply:
@johnsaigle I just added a commit that works for getting the last six characters of the string for comparison.

Reply:
I know, I'd prefer not to use magic numbers though.